clients/src/main/resources/common/message/DescribeClientQuotasRequest.json

```diff
@@ -16,7 +16,7 @@
 {
   "apiKey": 48,
   "type": "request",
-  "listeners": ["zkBroker", "broker"],
+  "listeners": ["zkBroker"],
```

Comment:
describeClientQuotas is implemented for kip-500 already (on the broker)

Reply:
I was going by this: https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/KafkaApis.scala#L3026.

Reply:
Hmm... looking at trunk, it looks like there are some changes from the KIP-500 branch that didn't make it over. `handleDescribeClientQuotasRequest` did work in the kip-500 branch.  Anyway, at this point, I think we should just disable it and put client quotas support on the TODO list for the next release (sigh)